libs/sql-schema-describer/src/mssql.rs

```diff
@@ -447,8 +444,6 @@ impl SqlSchemaDescriber {
                 ordinal_position
         "#;
 
-        debug!("describing table foreign keys, SQL: '{}'", sql);
```

Comment:
I assume these `debug` statements have been replaced by the `tracing-log` feature in quaint?

Reply:
Yes, it's the second commit (with very short explanation in commit description). The idea is that quaint already logs this, so it's unnecessarily verbose to log it again.